### PR TITLE
lokman_homework1

### PR DESCRIPTION
**Pull request by lokman1984**

### CODE REVIEW

**Comment by lokman1984** [comment]
homework1_week1

File: Week1/homework/step1.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+greeting = "Hallo, Wereld !"; //Nederlands
+console.log(greeting);
+greeting = "Hej ,världen!"; // Swidish
+console.log(greeting);
+greeting = "Hola, mundo!"; // Spanish
+console.log(greeting);
+greeting = "Merhaba, shabaaaab ^_^!"; // Arabic
+console.log(greeting);
+greeting = "Salut, le monde!"; // French
+console.log(greeting);
```
Comment (unmeshvrije) [inline Week1/homework/step1.js]: It is okay to reuse the same variable. But why not just use console.log("Namaste World")
If you want to save each language greeting in a different variable, that would be nicer.

File: Week1/homework/step2.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+"use strict";
+console.log('I\'m awesome'); // added escape backslash before the symbol
```
Comment (unmeshvrije) [inline Week1/homework/step2.js]: nice explanatory comments!

File: Week1/homework/step3.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+"use strict";
+let x;
+console.log("the value of my variable x will be a number");
+x = 1;
+console.log(x);
+x ++;
```
Comment (unmeshvrije) [inline Week1/homework/step3.js]: what is ++ ?
I don't think that was part of the assignment

File: Week1/homework/step4.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+let myString = "Lokman Dlimi";
+console.log("the value of my string will be:" + typeof myString);
+console.log( myString);
+myString = 34;
```
Comment (unmeshvrije) [inline Week1/homework/step4.js]: why not use another string as asked in the assignment ?

File: Week1/homework/step8.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  console.log("firstnumber same fourarray");
+}else{
+  console.log("firstnumber different fourarray");
+};
+if(typeof secondString === typeof thirdObjects){
+  console.log("secondstring same thirdobject")
+}else{
+  console.log("secondstring different thirdobject")
+}
+
```
Comment (unmeshvrije) [inline Week1/homework/step8.js]: please read the assignment again. we asked to use two strings and two objects

File: Week1/homework/step9.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+"use strict";
+let x = 7;
+console.log("The initial value of x is " + x);
+x = x % 3;
+console.log("When 7 is divided by 3 the return in whole numbers is 2");
+console.log("The remainder is " + x);
+
+let y = 21;
+console.log("The initial value of y is " + y);
+console.log("When 21 is divided by 3 the return in whole numbers is 5");
```
Comment (unmeshvrije) [inline Week1/homework/step9.js]: the console log is not right,
when 21 is divided by 3, the quotient is 7 not 5. (**return in whole number** is called quotient in Mathematical language)

File: Week1/homework/step3.js (hunk 3 above)
Comment (lokman1984) [reply Week1/homework/step3.js]: i tried to give an advanced solution, sorry

**Comment by lokman1984** [comment]
last update

**Pull request closed without merging**